sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/TelemetryItemTests.cs

```diff
@@ -20,6 +20,9 @@ public class TelemetryItemTests
         private const string ActivitySourceName = "TelemetryItemTests";
         private const string ActivityName = "TestActivity";
 
+        private const string RequestActivitySourceName = "Microsoft.AspNetCore";
+        private const string RequestActivityName = "Microsoft.AspNetCore.Hosting.HttpRequestIn";
```

Comment:
This could be removed if tests are removed that enforce the `TraceHelper.GetOperationNameV2()` and `TraceHelper.GetOperationName()` logic above.